mesheryctl/internal/cli/root/system/start.go

```diff
@@ -204,40 +204,39 @@ func start() error {
 		}
 
 		//changing the port mapping in docker compose
```

Comment:
This comment becomes obsolete now, isnt it ?
We are programmatically reading the port mapping from helpers.go rather than the docker compose file.